doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -41,23 +41,22 @@ jupyter:
 
 ## Local connection to a Workbench Server
 
-Execute the steps below to link PyWorkbench with a local Ansys Workbench session:
+Execute the steps below to link PyWorkbench with a local session:
+
 - Initiate Ansys Workbench
 - Input StartServer() in the Workbench Command Window
 - Utilize the given port number to link PyWorkbench with the server
```

Comment:
```suggestion
- Initiate Ansys Workbench.
- Enter the``StartServer()`` method in the Workbench command window.
- Use the given port number to link PyWorkbench with the server.
```
I don't see code for doing the preceding points.  Is this intentional?

Reply:
Yes, code is not required here.